Source/DafnyLanguageServer/Language/DiagnosticErrorReporter.cs

```diff
@@ -55,15 +58,18 @@ public class DiagnosticErrorReporter : ErrorReporter {
           }
         }
       }
+
+      var uri = GetDocumentUriOrDefault(error.Tok);
+      var diagnostic = new Diagnostic {
+        Severity = DiagnosticSeverity.Error,
+        Message = error.Msg,
+        Range = error.Tok.GetLspRange(),
+        RelatedInformation = relatedInformation,
+        Source = VerifierMessageSource.ToString()
+      };
       AddDiagnosticForFile(
-        new Diagnostic {
-          Severity = DiagnosticSeverity.Error,
-          Message = error.Msg,
-          Range = error.Tok.GetLspRange(),
-          RelatedInformation = relatedInformation,
-          Source = VerifierMessageSource.ToString()
-        },
-        GetDocumentUriOrDefault(error.Tok)
+        diagnostic,
+        uri
```

Comment:
FWIW, I'm not sure this small refactoring helps; it's convenient to see values in context.